File: CS/Q445945/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;

namespace Q445945
{
    public partial class Form1 : Form
    {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click_1(object sender, EventArgs e) {
            anotherColorPickEdit1.RemoveColorByName("Web", "Transparent");
        }

        private void simpleButton2_Click_1(object sender, EventArgs e) {
            anotherColorPickEdit1.RemoveColorsByNames("Web", new List<string>() { "Black", "Red" });
        }
    }
}

File: VB/Q445945/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms

Namespace Q445945
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub simpleButton1_Click_1(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
            anotherColorPickEdit1.RemoveColorByName("Web", "Transparent")
        End Sub

        Private Sub simpleButton2_Click_1(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton2.Click
            anotherColorPickEdit1.RemoveColorsByNames("Web", New List(Of String)() From {"Black", "Red"})
        End Sub
    End Class
End Namespace
